src/btree/bt_handle.c

```diff
@@ -75,6 +75,8 @@ __wt_btree_open(WT_SESSION_IMPL *session, const char *op_cfg[])
      */
     WT_RET(__btree_clear(session));
     memset(btree, 0, WT_BTREE_CLEAR_SIZE);
+    btree->evict_pos = WT_NPOS_INVALID;
+    btree->evict_ref_saved = NULL;
```

Comment:
In theory we can move this field above `evict_ref` so that the previous call to memset 0 clears it. It's a shame we can't do that with evict_pos but I understand why.

Reply:
hmm I don't really see why memset is not in there.

Reply:
I would actually create a __wt_btree_pos_clear function and call it from both `__btree_clear` and [vict_lru.c](https://github.com/wiredtiger/wiredtiger/pull/10924/files#diff-fba173dbad7997f847117319e1939715f1969181ab2937508a2c9303b201f017), that way we don't duplicate the logic.

Reply:
added a call to the clear function